src/climate_learn/models/modules/utils/metrics.py

```diff
@@ -2,11 +2,10 @@
 import torch
 from scipy import stats
 
-
 ### Training loss
 
 
-def mse(pred, y, vars, lat=None, log_postfix=""):
+def mse(pred, y, vars, lat=None, log_postfix="", splice_out_variables = -1):
```

Comment:
What is `splice_out_variables` doing?

Reply:
for iterative models, in the intermediate steps we predict all the 36 output vars, where as in the final test step we predict only the required 3 output vars. This splice_out_variables is used in the evaluation step to compute the final metrics on the 3 output vars